projects/storefrontlib/src/lib/ui/layout/header/header.component.scss

```diff
@@ -2,83 +2,124 @@
 @import 'functions';
 @import 'mixins';
 
-//header color variables
-$cx-header-background-color: 'dark' !default;
-$cx-header-color: 'inverse' !default;
-$cx-header-separator-color: 'secondary' !default;
-
-//header variables
-$cx-header-skipper-position: absolute !default;
-$cx-header-trbl-margin: 0 auto !default;
-$cx-header-bar-height: none !default;
-$cx-header-separator-position: absolute !default;
-$cx-header-separator-position-top: 43px !default;
-$cx-header-separator-position-left: 0 !default;
-$cx-header-separator-border-width: 1px !default;
-$cx-header-separator-width: 100vw !default;
-$cx-header-height-tablet: 90px !default;
-$cx-header-height-mobile: 70px !default;
-$cx-header-trbl-padding-tablet: 1.15rem 0 0 0 !default;
-$cx-header-trbl-padding-mobile: 0.5rem 0 0 0 !default;
+$space: 8px;
 
 :host {
-  @include reset;
-  @include var-color('background-color', $cx-header-background-color);
-  @include var-color('color', $cx-header-color);
+  background-color: var(--cx-background-color, var(--cx-g-color-dark));
+  color: var(--cx-color, var(--cx-g-color-inverse));
+  display: block;
 }
 
-.cx-header {
-  position: relative;
-  margin: $cx-header-trbl-margin;
-  max-height: $cx-header-bar-height;
-  overflow: visible;
+header {
+  max-width: var(--cx-max-width, 1140px);
+  margin: auto;
 
-  @include media-breakpoint-up(lg) {
-    position: static;
-    @include make-container-max-widths;
+  display: flex;
+  flex-wrap: wrap;
 
-    &:before {
+  > * {
+    align-self: center;
+  }
+}
+
+.SiteContext,
+.SiteLinks {
+  flex: 50%;
+  padding: $space 0;
+  font-size: 0.75rem;
+  color: var(--cx-color, var(--cx-g-color-light));
+}
+
+.SiteLogo {
+  --cx-width: 95px;
+}
+
+.SearchBox {
+  margin: $space auto;
+}
+
+.NavigationBar {
+  flex: 100%;
+}
+
+@include media-breakpoint-up(lg) {
+  header {
+    padding: 0 $space;
+  }
+
+  .cx-hamburger {
+    display: none;
+  }
+
+  // a small line separator in the UI to separate out some of the controls
+  .SiteLinks {
+    text-align: right;
+
+    &:after {
+      border-top: 1px solid var(--cx-border-color, var(--cx-g-color-secondary));
```

Comment:
I was thinking maybe have like `var(--cx-border-top, 1px solid var(--cx-border-color, var(--cx-g-color-secondary))`. In case a client really wants to customize this.